pow-migration/src/monitor.rs

```diff
@@ -132,7 +132,7 @@ fn is_valid_ready_txn(
 /// Checks if the provided transaction meets the criteria in order to be
 /// considered a valid online-transaction
 fn is_valid_online_txn(txn: &TransactionDetails, block_window: &Range<u32>) -> bool {
-    Some("online".to_string()) == txn.data
+    Some(hex::encode(String::from("online"))) == txn.data
```

Comment:
Um guessing this is optimized by the compiler for being static?

Reply:
It's not, but it also doesn't really matter.

```suggestion
    Some(hex::encode("online")) == txn.data
```

```suggestion
    txn.data.as_ref().is_some_and(|hex| hex == "6f6e6c696e65") // "online"
```